Changes

```diff
@@ -602,65 +602,65 @@ OCaml 4.07
   non-regular files, like other Unix variants do.
   (Xavier Leroy)
 
-- MPR#6394, GPR#1425: fix fatal_error from Parmatch.get_type_path
+- MPR#6394, GPR#1425: fix fatal_error from Parmatch.get_type_path.
   (Virgile Prevosto, review by David Allsopp, Thomas Refis and Jacques Garrigue)
 
 * MPR#6604, GPR#931: Only allow directives with filename and at the beginning of
-  the line
+  the line.
   (Tadeu Zagallo, report by Roberto Di Cosmo,
    review by Hongbo Zhang, David Allsopp, Gabriel Scherer, Xavier Leroy)
 
 - MPR#7138, MPR#7701, GPR#1693: Keep documentation comments
-  even in empty structures and signatures
+  even in empty structures and signatures.
   (Leo White, Florian Angeletti, report by Anton Bachin)
 
 - MPR#7178, MPR#7253, MPR#7796, GPR#1790: Make sure a function
-  registered with "at_exit" is executed only once when the program exits
+  registered with "at_exit" is executed only once when the program exits.
   (Nicolás Ojeda Bär and Xavier Leroy, review by Max Mouratov)
 
-- MPR#7391, GPR#1620: Do not put a dummy method in object types
+- MPR#7391, GPR#1620: Do not put a dummy method in object types.
   (Thomas Refis, review by Jacques Garrigue)
 
 - PR#7660, GPR#1445: Use native Windows API to implement Unix.utimes in order to
   avoid unintended shifts of the argument timestamp depending on DST setting.
   (Nicolás Ojeda Bär, review by David Allsopp, Xavier Leroy)
 
-- MPR#7668: -principal is broken with polymorphic variants
+- MPR#7668: -principal is broken with polymorphic variants.
   (Jacques Garrigue, report by Jun Furuse)
 
 - MPR#7680, GPR#1497: Incorrect interaction between Matching.for_let and
-  Simplif.simplify_exits
+  Simplif.simplify_exits.
   (Alain Frisch, report and review by Vincent Laviron)
 
-- MPR#7682, GPR#1495: fix [@@unboxed] for records with 1 polymorphic field
+- MPR#7682, GPR#1495: fix [@@unboxed] for records with 1 polymorphic field.
   (Alain Frisch, report by Stéphane Graham-Lengrand, review by Gabriel Scherer)
 
 - MPR#7695, GPR#1541: Fatal error: exception Ctype.Unify(_) with field override
   (Jacques Garrigue, report by Nicolás Ojeda Bär)
 
-- MPR#7704, GPR#1564: use proper variant tag in non-exhaustiveness warning
+- MPR#7704, GPR#1564: use proper variant tag in non-exhaustiveness warning.
   (Jacques Garrigue, report by Thomas Refis)
 
 - MPR#7711, GPR#1581: Internal typechecker error triggered by a constraint on
-   self type in a class type
+   self type in a class type.
   (Jacques Garrigue, report and review by Florian Angeletti)
 
-- MPR#7712, GPR#1576: assertion failure with type abbreviations
+- MPR#7712, GPR#1576: assertion failure with type abbreviations.
   (Thomas Refis, report by Michael O'Connor, review by Jacques Garrigue)
 
-- MPR#7747: Type checker can loop infinitly and consumes all computer memory
+- MPR#7747: Type checker can loop indefinitely and consumes all computer memory
```

Comment:
Perhaps "infinitely", as "indefinitely" connotes that it might terminate.

Reply:
consumes should also be consume